OptimizelySDKCore/OptimizelySDKCoreTests/OptimizelyTest.m

```diff
@@ -382,6 +382,84 @@ - (void)testIsFeatureEnabledWithFeatureFlagEnabledAndUserIsBeingExperimented {
     [decisionServiceMock stopMocking];
 }
 
+// Should return false if the feature experiment variation’s `featureEnabled` property is true
```

Comment:
The code comment seems to have a little error.  Our design doc says:

Test that `isFeatureEnabled` returns false if the feature experiment variation’s `featureEnabled` property is false

The last word in the code comment should be "false", not "true".